embed/Makefile.am

```diff
@@ -43,6 +43,10 @@ libephyembed_la_SOURCES = \
 	ephy-file-monitor.h		\
 	ephy-find-toolbar.c		\
 	ephy-find-toolbar.h		\
+	ephy-fx-password-notification.c	\
```

Comment:
I think embed is probably not the right location for these files. The files in this directory should be needed by a particular browser tab (an `EphyEmbed`, which contains exactly one `EphyWebView`), but not the rest of the application. That's why the existing code that displays info bars is here -- the info bars are tied to a particular tab. But your notifications are global to the application.

These files could probably both go under src instead. Or you could put ephy-fx-password-notification under lib/widgets and ephy-notification-manager under src.